vignettes/example1.Rmd

```diff
@@ -116,54 +116,57 @@ nrow(posteriors)  # Size (number of rows)
+If we look at the documentation (`?sampling`) for the rstanarm `"sampling"` algorithm used by default in the model above, we can see several parameters that influence the number of posterior draws. By default, there are **4** `chains` (you can see it as distinct sampling runs), that each create **2000** `iter` (draws). However, only half of these iterations are kept, as half are used for `warm-up` (the convergence of the algorithm). Thus, the total is **`4 chains * (2000 iterations - 1000 warm-up) = 4000`** posterior draws. We can change that, for instance:
 
 ```{r message=FALSE, warning=FALSE, eval=FALSE}
 model <- stan_glm(Sepal.Length ~ Petal.Length, data=iris, chains = 2, iter = 1000, warmup = 250)
  
 nrow(insight::get_parameters(model))  # Size (number of rows)
 ```
+ 
 ```{r echo=FALSE, message=FALSE, warning=FALSE, comment=NA, echo=FALSE}
 junk <- capture.output(model <- stan_glm(Sepal.Length ~ Petal.Length, data=iris, chains = 2, iter = 1000, warmup = 250))
 nrow(insight::get_parameters(model))  # Size (number of rows)
 ```
 
-In this case, we indeed have **`2 chains * (1000 iterations - 250 warm-up) = 1500`** posterior draws. Let's keep our first model with the default setup.
-
+In this case, as would be expected, we have **`2 chains * (1000 iterations - 250 warm-up) = 1500`** posterior draws. However, let's keep our first model with the default setup.
 
 #### Visualizing the posterior distribution
 
-
-Now that we've understood where these values come from, let's look at them. We will start by visualizing the distribution of our parameter of interest, the effect of `Petal.Length`.
+Now that we've understood where these values come from, let's look at them. We will start by visualizing the posterior distribution of our parameter of interest, the effect of `Petal.Length`.
 
 
 ```{r message=FALSE, warning=FALSE}
 ggplot(posteriors, aes(x = Petal.Length)) +
-  geom_density(fill = "orange")
+  geom_density(fill = "wheat")
 ```
 
 This distribution represents the [probability](https://en.wikipedia.org/wiki/Probability_density_function) (the y axis) of different effects (the x axis). The central values are more probable than the extreme values. As you can see, this distribution ranges from about **0.35 to 0.50**, with the bulk of it being at around **0.41**.
 
 > **Congrats! You've just described your posterior distribution.**
 
-And this is at the heart of Bayesian analysis. We don't need *p*-values, *t*-values or degrees of freedom: **everything is there**, within this **posterior distribution**.
+And this is at the heart of Bayesian analysis. We don't need *p* values, *t* values or degrees of freedom: **everything is there**, within this posterior distribution.
```

Comment:
throughout most of the package, we spell *p*-values (I think it is the APA-like spelling), so I suggest keeping the `-`

Reply:
I'm happy to retain the `-` for consistency across the package, though I believe the APA does recommend no hyphen in 6th ed.

Reply:
> I believe the APA does recommend no hyphen in 6th ed.

If that is confirmed we will need to slowly remove them in all easystats (as I meticulously placed them everywhere) ^^

Reply:
(do you have a link where it says that?)

Reply:
![image](https://user-images.githubusercontent.com/16096044/59745813-26862f80-92a8-11e9-8673-9994ca71f93a.png)

I can't actually find an explicit reference in the guide, however, you can see here (APA Style Guide 6th Ed., p. 114) that the guide writes "_p_ value" without the hyphen. However, you do have to have the hyphen for "_t_-test". Nevertheless, I've removed in the commit as it's something we can do a find and replace for at a later date.

Reply:
thanks. Yeah, in that case for v3 we will remove the hyphen everywhere ☺️